kitsune/sumo/api_utils.py

```diff
@@ -298,51 +253,35 @@ def has_object_permission(self, request, view, obj):
         return user == creator
 
 
-def PermissionMod(cls, permissions):
+PermissionListSerializer = None
+
+
+def PermissionMod(field, permissions=[]):
```

Comment:
On the one hand, I'm pretty sure this doesn't get mutated. On the other hand, we have no way to guarantee that, so this makes it a potential problem in the future. I'll change it.